ratelimit/redis.go

```diff
@@ -77,31 +77,27 @@ func (c *clusterLimitRedis) measureQuery(format, groupFormat string, fail *bool,
 	c.metrics.MeasureSince(key, start)
 }
 
-func (c *clusterLimitRedis) startSpan(ctx context.Context, spanName string) func(bool) {
-	nop := func(bool) {}
+func parentSpan(ctx context.Context) (opentracing.Span, bool) {
 	if ctx == nil {
```

Comment:
Can the `ctx` be ever nil actually?

Reply:
in tests yes

Reply:
I think we can fix it later, because this will likely be a bigger refactoring. https://github.com/zalando/skipper/issues/2136